zerver/webhooks/radarr/doc.md

```diff
@@ -0,0 +1,24 @@
+Receive Radarr notifications in Zulip!
+
+1.  {!create-stream.md!}
+
+1.  {!create-bot-construct-url-indented.md!}
+
+1.  Go to your Radarr
+    dashboard. Click **Settings** and then **Connect**.
+    Click the **+** icon.
+
+1.  Select **Webhook**. Enter **Zulip** or any other name you wish in the
+    name field. Tick which scenarios you would like to be notified for. You
+    can enter tags if you would only like to notify for movies with a
+    specific tag.
+
+1.  Enter the **URL** constructed earlier in the **URL** field. Ensure the
+    **Method** field is set to **POST**. Ensure the **Username** and
+    **Password** field are empty.
+
+1.  Click **Save** and you should receive a test message.
```

Comment:
I would apply the same comments that I made on your Lidarr integration PR's docs #17847 here as well.